smartsim/_core/entrypoints/redis.py

```diff
@@ -179,6 +191,14 @@ def cleanup() -> None:
         action="store_true",
         help="Specify if this orchestrator shard is part of a cluster",
     )
+    parser.add_argument(
+        "+redirect_output",
+        action="store_true",
+        help=(
+            "Specify if stdout and stder of this script should be redirected. "
```

Comment:
obligatory typo nitpick! stder -> stderr

Reply:
Done in https://github.com/al-rigazzi/SmartSim/pull/7